resources/schemas/dbscripts/postgresql/targetedms-create.sql

```diff
@@ -2,27 +2,24 @@ CREATE VIEW targetedms.AuditLog AS
 (
 
 WITH RECURSIVE logTree as (
-    SELECT entryId
-         , entryHash
-         , versionid AS RunId
-         , parentEntryHash
+    SELECT e.entryId
+         , e.entryHash
+         , e.parentEntryHash
     FROM targetedms.AuditLogEntry e
-    WHERE versionid IS NOT NULL
+    WHERE e.parentEntryHash = '(null)'
```

Comment:
Yuck. I didn't know we stored this as a string instead of just using null.

On PanoramaWeb.org we have an audit log table already. Since we don't have an index on parentEntryHash, I worry about perf here. I propose that we do an outer join to the new junction table here so that we can do a similar IS NOT NULL on VersionId.

Reply:
You can't do an outer join in a CTE on PG.  I tried your suggestion before going to this one.  It is a head scratcher why we're using "(null)".  It's also used in CTEs building the audit log tree.  I can look at factoring that to use regular NULL but trying to not let this PR continue to grow.